media/public/include/IControl.h

```diff
@@ -89,15 +84,16 @@ class IControl
      * @brief Register new IControlClient. This method will hold a shared_ptr
      *        to the client until the destruction of the IControl object.
      *        At destruction the client will be unregistered and the
-     *        shared_ptr will be released
+     *        shared_ptr will be released. NOTE: this is different
```

Comment:
I dont think this comment should be in the public interface, IClientController isn't a public interface so the user wont necessarily know what it is.